homeassistant/components/dialogflow/config_flow.py

```diff
@@ -0,0 +1,13 @@
+"""Config flow for DialogFlow."""
+from homeassistant.helpers import config_entry_flow
+from .const import DOMAIN
+
+
+config_entry_flow.register_webhook_flow(
+    DOMAIN,
+    'Dialogflow Webhook',
+    {
+        'dialogflow_url': 'https://dialogflow.com/docs/fulfillment#webhook',
+        'docs_url': 'https://www.home-assistant.io/components/dialogflow/'
```

Comment:
Can we read this url from` manifest.json`?

Reply:
Probably. I think that it might still be that sometimes users want to have specific set up instructions instead of generic instructions. If so, for a future PR.